lib/command/build_image.rb

```diff
@@ -18,7 +18,7 @@ class BuildImage < Base
     def call # rubocop:disable Metrics/MethodLength
       ensure_docker_running!
 
-      dockerfile = config.current[:dockerfile] || "Dockerfile"
+      dockerfile = config.current(:dockerfile) || "Dockerfile"
```

Comment:
@rafaelgomesxyz how is this. change related?

Reply:
Let's handle this in a separate PR.